app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from functools import wraps
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    @wraps(func)
+    def wrapper(*args, **kwargs) -> Any:
+        cache_key = args + tuple(kwargs.items())
```

Comment:
You can ignore kwargs and use only *args, as kwargs are not expected to be passed to this decorator